x/dex/types/tick_liquidity_key.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+type LiquidityType int
+
+const (
+	LiquidityType_PoolReserves LiquidityType = iota
+	LiquidityType_LimitOrder
+)
+
+type TickLiquidityKey interface {
```

Comment:
doesn't look like this interface is every really being used. Was there a larger plan here?

Reply:
Yes, which was abandoned. The larger plan was to stop storing the properties that are in the key in both the key and the value but this would have been too big of a change so I stopped where I was.